src/Outsystems.SetupTools/Lib/PlatformSetup.ps1

```diff
@@ -166,12 +239,12 @@ function InstallDotNetCore([string]$Sources)
 {
     if($Sources)
     {
-        $installer = "$Sources\DotNetCore_2_WindowsHosting.exe"
+        $installer = "$Sources\DotNetCore_WindowsHosting.exe"
```

Comment:
We should move this to a constant in src/Outsystems.SetupTools/Lib/Constants.ps1